src/components/newcases.js

```diff
@@ -0,0 +1,114 @@
+import React, {useState, useEffect} from 'react';
+
+function RateOfNewCases(props) {
+  const [firstDayCases, setFirstDayCases] = useState(0);
+  const [firstDayDate, setFirstDayDate] = useState(0);
+  const [secondDayCases, setSecondDayCases] = useState(0);
+  const [secondDayDate, setSecondDayDate] = useState(0);
+  const [thirdDayCases, setThirdDayCases] = useState(0);
+  const [thirdDayDate, setThirdDayDate] = useState(0);
+  const [fourthDayCases, setFourthDayCases] = useState(0);
+  const [fourthDayDate, setFourthDayDate] = useState(0);
+  const [fifthDayCases, setFifthDayCases] = useState(0);
+  const [upArrow, setUpArrow] = useState(0);
+  const [downArrow, setDownArrow] = useState(0);
+
+  useEffect(() => {
+    parseData();
+    setUpArrow(String.fromCharCode(8593));
+    setDownArrow(String.fromCharCode(8595));
+  });
```

Comment:
Please put second argument for useEffect as []. Otherwise component will render infinitely.

Reply:
Yes it should render similar to the level.js. we want to load without refreshing page.

Reply:
No. it has other unintended consequences. 
```js
useEffect(() => {
  parseData();
}, [props.timeseries])
```
This is the correct thing to write.

Reply:
thanks .. I have changed this.